dhis-2/dhis-services/dhis-service-reporting/src/main/java/org/hisp/dhis/validation/ValidationRunContext.java

```diff
@@ -27,16 +27,14 @@
  */
 package org.hisp.dhis.validation;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
-import java.util.Queue;
-import java.util.Set;
+import static java.util.Objects.requireNonNull;
+
+import java.util.*;
```

Comment:
I've been told (by Morten H and others) that we don't do wildcard imports in DHIS2 code (though they are in some of my older code before I was told this). I've configured my IDE (IntelliJ) to not generate them.